mlir/lib/Dialect/Tosa/Transforms/TosaFolders.cpp

```diff
@@ -49,8 +49,7 @@ OffsetType indexToOffset(DimensionType shape, DimensionType index) {
   return offset;
 }
 
-SmallVector<int64_t> offsetToIndex(DimensionType shape,
```

Comment:
Can you please revert all reformatting changes from this file? Upstream typically doesn't accept them and it will make our fork diverge more from upstream, making upstreaming harder.